### PR TITLE
new feature with numbers

### PR DESCRIPTION
**Pull request by mazhuga**
Here is new feature

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
-# Git and Github Course
+# Git and 
+Github Course
```
Comment (nicjkkd) [inline readme.md]: Remove new line

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
 
-Crash course for Git
\ No newline at end of file
+Crash course for Git
+
+1
+2
+3
```
Comment (nicjkkd) [inline readme.md:8]: After this line should be line with num 4

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+5
+6
+7
+8
+9
+0
```
Comment (nicjkkd) [inline readme.md]: Zero should be first

**Review by nicjkkd — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (mazhuga) [reply readme.md]: done

**Review by nicjkkd — APPROVED** [review]

**Pull request merged**